docs/api/puppeteer.connectoptions.headers.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_label: ConnectOptions.headers
+---
+
+# ConnectOptions.headers property
+
+#### Signature:
+
+```typescript
+interface ConnectOptions {
+  headers?: Record<string, string>;
+}
+```
```

Comment:
Tried to add here something like:
```
## Remarks

**NOTE** Works only in Node.js environment.
```

But `npm run docs` rewrite my changes.

Reply:
The docs should go into code comments and our build re-generated markdown based on code comments. E.g.,

```
/**
* Headers to use for the web socket connection in Node 
*/
headers?: Record<string, string>;
```